collector/host_inventory.py

```diff
@@ -98,6 +100,7 @@ def worker(_: str, source_id: str, dest: str, b64_identity: str) -> None:
     prometheus_metrics.METRICS['posts'].inc()
     try:
         utils.retryable('post', dest, json=data, headers=headers)
+        utils.set_processed(account_id)
```

Comment:
This is a good spot to set the "processed" flag (after 'post', that is)

We could do this in `collector/topological_inventory.py` as well